mount.go

```diff
@@ -132,6 +144,35 @@ func mount(source, destination, fsType string, flags int, options string) error
 	return nil
 }
 
+// Allocate hugepages by writing to sysfs
+func allocateHugePages(options string) error {
+
+	agentLog.WithField("HugePagesOption", options).Info("HugePages option string")
+
+	//options eg "pagesize=2048,size=107374182"
+	opt := strings.Split(options, ",")
+	pageSizeStr := strings.TrimPrefix(opt[0], "pagesize=")
+	pageSize, err := strconv.ParseInt(pageSizeStr, 10, 64)
+	if err != nil {
+		return err
+	}
+	//sysfs entry is always of the form hugepages-${size}kB
+	//Ref: https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt
+	sysfsHugepagesPath := fmt.Sprintf("%s/hugepages-%skB/nr_hugepages", sysfsHugepagesPrefix, strconv.FormatInt((pageSize/1024), 10))
```

Comment:
Can you use `path.Join()` / `filepath.Join()` here?

Reply:
yeah sure